lib/modules/easterEgg.js

```diff
@@ -23,9 +25,27 @@ module.go = () => {
 			notificationID: 'konami',
 		});
 		setTimeout(() => $baconBit.addClass('makeitrain'), 500);
-	};
-	konami.iphone.load = () => {}; // nix touch support; occasional false positives on touchscreen laptops
-	konami.load();
+	}
+
+	konami.addEvent(document, "keydown", function (e, ref_obj) {
```

Comment:
At this point we might as well just drop konami.js and move the rest of the implementation (that we need) into this module.

We don't even use half of it anyways (mobile and IE support).